Content.Server/Nuke/NukeCodePaperSystem.cs

```diff
@@ -46,17 +48,36 @@ public bool SendNukeCodes(EntityUid station)
                 return false;
             }
 
-            var wasSent = false;
             var faxes = EntityManager.EntityQuery<FaxMachineComponent>();
+
+            string? centcomFaxAddress = null;
+            foreach (var fax in faxes)
+            {
+                if (!TryComp<MetaDataComponent>(fax.Owner, out var metadata) ||
+                    metadata.EntityPrototype?.ID != CentcomFaxPrototypeId ||
+                    !TryComp<DeviceNetworkComponent>(fax.Owner, out var device))
+                {
```

Comment:
Don't hardcode prototype IDs like this.

Reply:
I don't see any better solutions. Only add the `IsContcom` parameter to the component or maybe try to get a fax from the centcom grid.

Reply:
We are not merging it like this.
Do quite literally anything else, don't hardcode a check to the central command grid either.